data/features/mmex.json

```diff
@@ -4,8 +4,8 @@
     "features": [
         {  "name":"accounts",
             "icon": "fas fa-money-check",
-            "title": "Accounts & Currencies",
-            "text": "Wizard to simply create accounts and start to use MMEX. Multiple currencies for each account to have more flexibility."
+            "title": "Accounts and currencies",
+            "text": "Wizard to simply account creation to start using MMEX. Multiple currencies for each account to have more flexibility."
```

Comment:
1. "Wizard to simply create accounts and start to use MMEX."
2. "Wizard to simply account creation to start using MMEX."

In the first sentence, the verb "create" directly acts on "accounts," indicating the action of creating accounts. This structure is clear and concise.

In the second sentence, "simply account creation" is not grammatically correct. It appears to be attempting to combine "simply account creation" into a noun phrase, but it lacks the necessary structure for this to work effectively. It would be better to use "create accounts" as in the first sentence.

Reply:
Thank you for your comment. Yes, it is not grammatically correct.

Would changing "simply" to "simplify" work?
"Wizard to simplify account creation to start using MMEX."